main/createUiDefinition.json

```diff
@@ -16,14 +16,26 @@
               "value": "vnios-te-v820"
             },
             {
+              "label": "TE-V825",
```

Comment:
I am ok, but you should make sure that model names for SoT should really be diaplayed as "TE-x25".